extensions/amp-lightbox-viewer/0.1/service/lightbox-manager-impl.js

```diff
@@ -89,6 +89,13 @@ export class LightboxManager {
      * @private {number}
      */
     this.counter_ = 0;
+
+    /**
+     * If the lightbox group is a carousel, this object contains a
+     * mapping of the lightbox group id to the carousel element.
+     * @private {!Object<string, !Element>}
+     */
+    this.lightboxSourceCarousels_ = {};
```

Comment:
This might need to be a `src/object.js#map` to speed up lookups. `map` is prototypeless so failed lookups happen faster. In the AMP codebase I see we generally prefer `map`s and `map[key]` notation vs `{}` literals and `hasOwnProperty`.

Reply:
@jridgewell any guidance?

Reply:
Discussed this and yes you're right. This should happen for security reasons too, not just performance.